omod/src/main/java/org/openmrs/module/metadatasharing/web/controller/PublishController.java

```diff
@@ -226,7 +227,25 @@ public void getNewPackage(@RequestParam(required = false)
 								}
 							}
 						}
-					}
+						if(uuids == null && ids == null && modifiedSince != null) {
+							List<Object> items=Handler.getItems(type, includeRetired, null, null, null);
+							if (items != null) {
+								for(Object item: items ) {								
+									if (OpenmrsUtil.compareWithNullAsEarliest(Handler.getDateChanged(item),
```

Comment:
Can we modify this test to be if  "modifiedSince == null || OpenmrsUtil.compareWithNullAsEarliest...".  Then we should be able to get rid of the second block (lines 241 thru 248) entirely?

Reply:
@mogoodrich  I will make the changes shortly...

Reply:
Thanks @reagan-meant !